src/modules/renderer.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable linebreak-style */
+const renderTodo = (todoLiRow, clearSelected, dragger, todoFilter) => {
+  const list = document.getElementById('todo-list');
+  list.innerHTML = '';
+  todoLiRow.sort((a, b) => a.index - b.index); // sort tasks by index
+  if (todoLiRow.some((element) => element.completed)) {
+    clearSelected.disabled = false;
+  } else {
+    clearSelected.disabled = true;
+  }
+  todoLiRow.forEach((element, index) => { // map, foreach
+    const task = todoLiRow[index];
+    const listItem = document.createElement('li');
+    const delicon = document.createElement('i');
+    // creating the drag button
+    const drag = document.createElement('i');
+    listItem.className = 'list-element';
+
+    // create a checkbox and label for each task
+    const checkTodo = document.createElement('div');
+    checkTodo.className = 'chk-todo';
+    const todo = document.createElement('input');
+    todo.className = 'todo';
+    todo.value = task.description;
+    todo.addEventListener('focusin', () => {
+      delicon.style.display = 'inline';
+      drag.style.display = 'none';
+    });
+    todo.addEventListener('focusout', () => {
+      setTimeout(() => {
+        delicon.style.display = 'none';
+        drag.style.display = 'inline';
+      }, 3000);
+    });
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = task.completed;
+    checkbox.addEventListener('change', () => dragger(task, checkbox, todo, todoLiRow, clearSelected));
```

Comment:
- [ ] when I click the checkbox, the iscompleted property is not being updated in the local storage, kindly fix that
![Screenshot from 2023-04-09 00-02-00](https://user-images.githubusercontent.com/66097371/230746049-aa2abcc7-1f8d-4ad7-9a4b-0742ff758c90.png)